### PR TITLE
Added minutes on mie meeting

### PR DESCRIPTION
**Pull request by joelostblom**
@mbonsma check that this represents what we heard.

### CODE REVIEW

File: course/meetings/mie-matthew-mackay-2017-03-13.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- He wants them to have minimum 3 year programming.
+- Mechs are very different, avoid anything computer or electrical engineering.
+    - But usually have second thoughts on this. Currently no programming from second year to 4th year fall term. Different programming in 4th year matlab etc, microcontrollers.
+- He has been pushing for a core mech programming course.
+- It could be more effective if shared between mech and ind, but difficult.
+- Programming on the mech side hard sell, but they want to get it.
+- Replace and update numerical methods course. That is more of an intro to programming course.
+ RQM is closer to the IND side since they do more research already.  There is also a financial engineering that fits in.
+- Mech side still has data collection and data gathering. Students might not be interested in research methods, since they don't know they need it until 3 rd year research project. Might have to make something more targeting to a specific project. Prof diller is a potential contact.
+- Lots of faculty that like the status quo, we need to
```
Comment (QuLogic) [inline course/meetings/mie-matthew-mackay-2017-03-13.md]: Unfinished sentence?

File: course/meetings/mie-matthew-mackay-2017-03-13.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+
+---
+contact: Matthew Mackay
+department: Mechanical and Industrial Engineering
+uoftcoders: Madeleine, Joel
+date: 2017-03-13
+---
+
+# Summary
+- His ideal course would be somewhat engineering specific and attractive to both mechanical and industrial engineers. He seemed more interesting in uniting these two engineering streams than bridging gaps to other departments.
```
Comment (mbonsma) [inline course/meetings/mie-matthew-mackay-2017-03-13.md]: interesting -> interested

File: course/meetings/mie-matthew-mackay-2017-03-13.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - If there was a course that bridged the interests of the Mechanical and Industrial streams, that would be very attractive to the department.
+- Want more programming in general for his undergrads.
+    - Currently a big gap between taking a few courses in first year and then basically nothing until fourth year.
+        - At least for Mechanical engineers where programming is a hard sell, the industrial stream has decent programming experience.
+    - Currently mostly dependent on student electives.
+- The RQM could potentially replace and update their numerical methods course.
```
Comment (mbonsma) [inline course/meetings/mie-matthew-mackay-2017-03-13.md]: @joelostblom, do you remember if this was a third or fourth-year course?

File: course/meetings/mie-matthew-mackay-2017-03-13.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- RQM course would be closer to industrial engineering where they do more research.
+    - There is also financial engineering that could be interesting
+    - Environmental engineering could be a match with EEB?
+- Grad students are allowed to teach, but they mostly get professional engineers.
+- Will put us in contact with some of their teaching grad students for tips.
+- There might be some resistance from higher levels that tend to like the status quo.
+
```
Comment (mbonsma) [inline course/meetings/mie-matthew-mackay-2017-03-13.md]: Suggested to contact [Bruno Korst](https://www.ece.utoronto.ca/undergraduates/hardware-computer-labs/communications-lab/) in ECE - he's a lab manager with clout, might be interested in the course.

File: course/meetings/mie-matthew-mackay-2017-03-13.md (hunk 1 above)
Comment (joelostblom) [inline course/meetings/mie-matthew-mackay-2017-03-13.md]: Probably an unfinished thought at the time, removing.

File: course/meetings/mie-matthew-mackay-2017-03-13.md (hunk 3 above)
Comment (joelostblom) [inline course/meetings/mie-matthew-mackay-2017-03-13.md]: Seems to be 3rd year http://coursefinder.utoronto.ca/course-search/search/courseInquiry?methodToCall=start&viewId=CourseDetails-InquiryView&courseId=MIE334H1S20171

**Pull request merged**